src/tasks/TaskItem.js

```diff
@@ -1,8 +1,16 @@
-function TaskItem({ deleteCompletedTask, obj, item }) {
+import DataContext from "../store/DataContext";
+
+import { useContext } from "react";
+
+//gets task object as obj and task name as item from TaskList
+function TaskItem({ obj, item }) {
```

Comment:
For taking props, use prop [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)

```suggestion
function TaskItem(props) {
const { obj, item } = props;
```

Reply:
Props destructuring applied